mmaction/datasets/pipelines/augmentations.py

```diff
@@ -134,38 +134,58 @@ def __init__(self, transforms):
                 [self.imgaug_builder(t) for t in self.transforms])
 
     def default_transforms(self):
-        """Default transforms for imgaug."""
+        """Default transforms for imgaug.
+
+        Implement RandAugment by imgaug.
+        Plase visit `https://arxiv.org/abs/1909.13719` for more information.
+
+        Augmenters and hyper parameters are borrowed from the following repo:
+        https://github.com/tensorflow/tpu/blob/master/models/official/efficientnet/autoaugment.py # noqa
+
+        Miss one augmenter ``SolarizeAdd`` since imgaug doesn't support this.
+
+        Returns:
+            dict: The constructed RandAugment transforms.
+        """
+        # RandAugment hyper params
+        n, m, max_level = 2, 9, 10
+        cur_level = 1.0 * m / max_level
```

Comment:
It seems that `n, m, max_level` is used only in some certain transform. Could you rename the variable with a more meaningful name?